### PR TITLE
[admin-v1] Fixes tables generation, tables creation, tables editing

### PR DESCRIPTION
**Pull request by mike-lvov**
Fixes https://github.com/sparkletown/internal-sparkle-issues/issues/1478

### CODE REVIEW

File: src/components/molecules/TablesUserList/TablesUserList.tsx [hunk 1]
```diff
@@ -198,15 +177,6 @@ export const TablesUserList: React.FC<TablesUserListProps> = ({
...
   );
 
-  const emptyTables = useMemo(
-    () =>
-      tables.filter((table) => !usersSeatedAtTables[table.reference].length),
-    [tables, usersSeatedAtTables]
-  );
-
-  const canStartTable =
-    emptyTables.length <= ALLOWED_EMPTY_TABLES_NUMBER && !isSeatedAtTable;
```
Comment (sunny-viktoryia) [inline src/components/molecules/TablesUserList/TablesUserList.tsx:208]: is the logic changed intentionally? As far as I see, the conditions of adding tables changed.

**Review by sunny-viktoryia — APPROVED** [review]
Looks good to me. 🙃

File: src/components/molecules/TablesUserList/TablesUserList.tsx (hunk 1 above)
Comment (mike-lvov) [reply src/components/molecules/TablesUserList/TablesUserList.tsx:208]: Actually, I got an idea on how to do it even better

Comment (mike-lvov) [reply src/components/molecules/TablesUserList/TablesUserList.tsx]: @sunny-viktoryia so, I reworked the bug fix a bit to preserve the previous logic in cc5b02bd685dd6ea627055a4eb3a2b8fcebdb3b4

File: src/pages/FriendShipPage/FriendShipPage.tsx [hunk 2]
```diff
@@ -53,11 +53,12 @@ export const FriendShipPage: React.FunctionComponent = () => {
             }`}
           >
             <TablesUserList
-              venueName={venue.name}
+              venue={venue}
               setSeatedAtTable={setSeatedAtTable}
               seatedAtTable={seatedAtTable}
               TableComponent={TableComponent}
               customTables={FRIENDSHIP_CUSTOM_TABLES}
+              defaultTables={FRIENDSHIP_CUSTOM_TABLES}
```
Comment (sunny-viktoryia) [inline src/pages/FriendShipPage/FriendShipPage.tsx:61]: is this FriendshipPage used anywhere? it feels like it's a legacy one.

**Review by sunny-viktoryia — APPROVED** [review]
the code looks fine. 🙃 
haven't tested the UI.

File: src/pages/FriendShipPage/FriendShipPage.tsx (hunk 2 above)
Comment (mike-lvov) [reply src/pages/FriendShipPage/FriendShipPage.tsx:61]: Nope, it's legacy

**Pull request merged**